crates/ruff/resources/test/fixtures/pylint/logging_E1206.py

```diff
@@ -0,0 +1,12 @@
+import logging
```

Comment:
Nit: can we name these to match the rule? I think that's the convention we've been using for Pylint (e.g., `logging_too_few_args.py`)